libs/designsystem/reorder-list/src/reorder-list.component.scss

```
@@ -61,6 +61,10 @@ ion-backdrop {
   border-bottom: 20px transparent solid;
 }
 
+.outer-group-last {
+  border-bottom: transparent solid;
+}
```

Consider simplifying:
```suggestion
ion-reorder-group > div {
  border-bottom: 20px transparent solid;
  
  &:last-child {
    border-bottom: transparent solid;
  }
}
```